### DISCUSSION TITLE
第3节课笔记(21班)

### DISCUSSION THREAD

**Discussion by mattheliu** [InternLM/Tutorial#223] [笔记]
请提交笔记链接 Please submit the link to the notes.

**Comment by JinProton** [comment]
https://zhuanlan.zhihu.com/p/677933496

**Comment by yunhuisong2023** [comment]
https://reflective-value-ac9.notion.site/3-InternLM-LangChain-93f6e16c46d44cfdbacd0032428332aa

**Comment by HaroldChen19** [comment]
https://github.com/HaroldChen19/InternLM-Boot-Camp/blob/master/Lesson_3/Note.md

**Comment by jayjayhust** [comment]
https://github.com/jayjayhust/InternLM-Tutorial/blob/main/Lesson3.md

**Comment by water2silver** [comment]
https://zhuanlan.zhihu.com/p/679036637
笔记和作业都是这个连接

**Comment by cqu-student** [comment]
https://zhuanlan.zhihu.com/p/677605223

**Comment by TonyTnT** [comment]
https://zhuanlan.zhihu.com/p/679576237

**Comment by YongXie66** [comment]
https://github.com/YongXie66/InternLM_tutorial_copy/blob/main/notes/Lesson3.pdf

**Comment by mattheliu** [comment]
https://github.com/mattheliu/gitblog/issues/3
